notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "metadata": {},
+   "source": [
+    "#### :# -- Add comment to logfile\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### :show -- Page the full output of  the  previous  deblend  and  equivalent         width measurements.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "\n",
+    "`specviz` will meet the logging need in two ways:\n",
+    "\n",
+    "1. For notebook-oriented workflows, the notebook itself can the \"save file\" - e.g., computing the equivalent width using a `SpectrumRegion` that's explicitly specified already records the result for reproducibility. [ALREADY IMPLEMENTED]\n",
+    "2. For desktop tool workflows (or notebook workflows in part using plugins), a log should be available to track operations. This could/should use the standard Python logging functionality, but to reproduce the specific needs of SPLOT, plugins should have a way to explicitly say \"log my results\". This log file should be accessible in the UI which would then have a \"save\" option for the user to record this if they want to."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Labels: various forms of labeling\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "    Labels:\n",
+    "\n",
+    "        :label <label> <format>\n",
+    "            Add a label at the cursor position.\n",
+    "        \n",
+    "        :mabove <label> <format>\n",
+    "            Add  a  tick mark and label above the spectrum at the cursor\n",
+    "            position.\n",
+    "        \n",
+    "        :mbelow <label> <format>\n",
+    "            Add a tick mark and label below the spectrum at  the  cursor\n",
+    "            position.\n",
+    "        \n",
+    "        The  label  must  be  quoted  if  it  contains  blanks.  A label\n",
+    "        beginning with % (i.e. %.2f) is treated as a format  for  the  x\n",
+    "        cursor  position.   The  optional  format is a gtext string (see\n",
+    "        help on \"cursors\").   The  labels  are  not  remembered  between\n",
+    "        redraws."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "\n",
+    "`bqplot` and `glue`/`glupyter` have mechanisms for setting labels.  It might be a bonus to add convenient accessors to make setting such labels easier along the lines of the cell below (but not this is not critical functionality):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.set_flux_label('My Nobel Prize-winning spectrum', include_unit=False)\n",
+    "specviz.set_spec_label('My special name for the spectral axis', include_unit=True)"
+   ]
```

Comment:
Labels also need a location and size specification.

Reply:
Agreed, but that was meant to be encompased in the words just above here.  Will try to clarify.